contrib/blueflood-docker/configs/blueflood.cdl

```diff
@@ -200,111 +200,6 @@ CREATE TABLE IF NOT EXISTS metrics_full (
-  compaction={'class': 'SizeTieredCompactionStrategy'} AND
-  compression={'chunk_length_kb': '64', 'sstable_compression': 'SnappyCompressor'};
-
-CREATE TABLE IF NOT EXISTS metrics_histogram_20m (
-  key text,
-  column1 bigint,
-  value blob,
-  PRIMARY KEY ((key), column1)
-) WITH COMPACT STORAGE AND
-  bloom_filter_fp_chance=0.010000 AND
-  caching='KEYS_ONLY' AND
-  comment='' AND
-  dclocal_read_repair_chance=0.000000 AND
-  gc_grace_seconds=864000 AND
-  index_interval=128 AND
-  read_repair_chance=0.100000 AND
-  replicate_on_write='true' AND
-  populate_io_cache_on_flush='false' AND
-  default_time_to_live=0 AND
-  speculative_retry='NONE' AND
-  memtable_flush_period_in_ms=0 AND
-  compaction={'class': 'SizeTieredCompactionStrategy'} AND
-  compression={'chunk_length_kb': '64', 'sstable_compression': 'SnappyCompressor'};
-
-CREATE TABLE IF NOT EXISTS metrics_histogram_240m (
-  key text,
-  column1 bigint,
-  value blob,
-  PRIMARY KEY ((key), column1)
-) WITH COMPACT STORAGE AND
-  bloom_filter_fp_chance=0.010000 AND
-  caching='KEYS_ONLY' AND
-  comment='' AND
-  dclocal_read_repair_chance=0.000000 AND
-  gc_grace_seconds=864000 AND
-  index_interval=128 AND
-  read_repair_chance=0.100000 AND
-  replicate_on_write='true' AND
-  populate_io_cache_on_flush='false' AND
-  default_time_to_live=0 AND
-  speculative_retry='NONE' AND
-  memtable_flush_period_in_ms=0 AND
-  compaction={'class': 'SizeTieredCompactionStrategy'} AND
-  compression={'chunk_length_kb': '64', 'sstable_compression': 'SnappyCompressor'};
-
-CREATE TABLE IF NOT EXISTS metrics_histogram_5m (
-  key text,
-  column1 bigint,
-  value blob,
-  PRIMARY KEY ((key), column1)
-) WITH COMPACT STORAGE AND
-  bloom_filter_fp_chance=0.010000 AND
-  caching='KEYS_ONLY' AND
-  comment='' AND
-  dclocal_read_repair_chance=0.000000 AND
-  gc_grace_seconds=864000 AND
-  index_interval=128 AND
-  read_repair_chance=0.100000 AND
-  replicate_on_write='true' AND
-  populate_io_cache_on_flush='false' AND
-  default_time_to_live=0 AND
-  speculative_retry='NONE' AND
-  memtable_flush_period_in_ms=0 AND
-  compaction={'class': 'SizeTieredCompactionStrategy'} AND
-  compression={'chunk_length_kb': '64', 'sstable_compression': 'SnappyCompressor'};
-
-CREATE TABLE IF NOT EXISTS metrics_histogram_60m (
-  key text,
-  column1 bigint,
-  value blob,
-  PRIMARY KEY ((key), column1)
-) WITH COMPACT STORAGE AND
-  bloom_filter_fp_chance=0.010000 AND
-  caching='KEYS_ONLY' AND
-  comment='' AND
-  dclocal_read_repair_chance=0.000000 AND
-  gc_grace_seconds=864000 AND
-  index_interval=128 AND
-  read_repair_chance=0.100000 AND
-  replicate_on_write='true' AND
-  populate_io_cache_on_flush='false' AND
-  default_time_to_live=0 AND
-  speculative_retry='NONE' AND
-  memtable_flush_period_in_ms=0 AND
-  compaction={'class': 'SizeTieredCompactionStrategy'} AND
-  compression={'chunk_length_kb': '64', 'sstable_compression': 'SnappyCompressor'};
-
 CREATE TABLE IF NOT EXISTS metrics_locator (
```

Comment:
@shintasmith I was tuning Blueflood docker code and came across this change. This change can cause BF container  failure as it is using the latest released jar of BF on github, which does contain Histogram code. I would highly recommend releasing a new JAR with all these changes.